src/space/dx11/perspective_handler.rs

```diff
@@ -57,17 +57,20 @@ impl PerspectiveHandler {
         }
     }
 
-    pub fn create_constant_buffer(device: &ID3D11Device) -> anyhow::Result<ID3D11Buffer> {
+    pub fn create_constant_buffer(device: &ID3D11Device, initial: &PerspectiveData) -> anyhow::Result<ID3D11Buffer> {
         let constant_buffer_desc = D3D11_BUFFER_DESC {
-            ByteWidth: size_of::<PerspectiveData>() as u32,
+            ByteWidth: size_of::<PerspectiveData>().next_multiple_of(16) as u32,
             Usage: D3D11_USAGE_DEFAULT,
             BindFlags: D3D11_BIND_CONSTANT_BUFFER.0 as u32,
             CPUAccessFlags: 0,
             MiscFlags: 0,
             StructureByteStride: 0,
         };
 
-        let constant_subresource_data = D3D11_SUBRESOURCE_DATA::default();
+        let constant_subresource_data = D3D11_SUBRESOURCE_DATA {
+            pSysMem: initial as *const PerspectiveData as *const _,
+            .. D3D11_SUBRESOURCE_DATA::default()
+        };
```

Comment:
this change isn't relevant and could be its own PR, it's just a fix for an issue that prevented the experimental space-kat-renderer from ever actually working on my machine (win10+nvidia drivers)